iac-extensions/docker/src/test/resources/checks/BuilderSandboxCheck/BuilderSandboxCheck.dockerfile

```diff
@@ -0,0 +1,14 @@
+# syntax=docker/dockerfile:1-labs
+FROM ubuntu:22.04
+
+# Noncompliant@+1 {{Make sure that disabling the builder sandbox is safe here.}}
+RUN --security=insecure cat /proc/self/status > /output.txt
+#   ^^^^^^^^^^^^^^^^^^^
+
+# Noncompliant@+1
+RUN --security=insecure ["cat", "/proc/self/status > /output.txt"]
+
+RUN --security=sandbox cat /proc/self/status > /output.txt
+RUN cat /proc/self/status > /output.txt
+RUN --network=none pip install --find-links wheels mypackage
```

Comment:
Possible other usecase :
```
--SECURITY=insecure
--security=INSECURE
ARG security=insecure
--security=$security
```
Not necessary, I let you decide.

Reply:
Good point